dff/script/core/script.py

```diff
@@ -24,7 +25,7 @@
 Context = ForwardRef("Context")
 
 
-class Node(BaseModel, extra=Extra.forbid):
+class Node(BaseModel, extra="forbid"):
```

Comment:
In this file too:

I think we should use the same way for model configuration for all our models:
either `ConfigDict` or class kwargs.

Reply:
moved to kwargs